python/GafferSceneUI/CameraUI.py

```diff
@@ -327,140 +321,66 @@
-
-			"description",
-			"""
-			Override the "overscan" render option.  Adds extra pixels to the sides of the rendered
-			image.  This can be useful when camera shake or blur will be added as a post process.
-			This plug just enables overscan as a whole - use the overscanTop, overscanBottom,
-			overscanLeft and overscanRight plugs to specify the amount of overscan on each side of
-			the image.
-			""",
-
-		],
-
-		"renderSettingOverrides.overscanTop" : [
-
-			"description",
-			"""
-			Override the "overscanTop" render option.  The amount of overscan at the top of the
-			image. Specified as a 0-1 proportion of the original image height.
-			""",
-
-		],
-
-		"renderSettingOverrides.overscanBottom" : [
-
-			"description",
-			"""
-			Override the "overscanBottom" render option.  The amount of overscan at the bottom of the
-			image. Specified as a 0-1 proportion of the original image height.
-			""",
-
-		],
-
-		"renderSettingOverrides.overscanLeft" : [
-
-			"description",
-			"""
-			Override the "overscanLeft" render option.  The amount of overscan at the left of the
-			image. Specified as a 0-1 proportion of the original image height.
-			""",
-
-		],
-
-		"renderSettingOverrides.overscanRight" : [
-
-			"description",
-			"""
-			Override the "overscanRight" render option.  The amount of overscan at the right of the
-			image. Specified as a 0-1 proportion of the original image height.
-			""",
-
-		],
-
-		"renderSettingOverrides.cropWindow" : [
-
-			"description",
-			"""
-			Override the "renderCropWindow" render option. Limits the render to a region of the
-			image. The rendered image will have the same resolution as usual, but areas outside
-			the crop will be rendered black. Coordinates range from 0,0 at the top left of the
-			image to 1,1 at the bottom right. The crop window tool in the viewer may be used to
-			set this interactively.
-			""",
+}
+
+## Grab the render overrides from the StandardOptions plug metadata,
+# change their key prefix, but keep their descriptions
+__overridesMetadata = {}
+for key, value in GafferSceneUI.StandardOptionsUI.plugsMetadata.iteritems() :
+
+	# Only grab the metadata for the plugs that the Camera node
+	# overrides
+	if key in [
+		"options.filmFit",
+		"options.shutter",
+		"options.renderResolution",
+		"options.pixelAspectRatio",
+		"options.resolutionMultiplier",
+		"options.overscan",
+		"options.overscanLeft",
+		"options.overscanRight",
+		"options.overscanTop",
+		"options.overscanBottom",
+		"options.cropWindow",
+		"options.depthOfField",
+	]:
+		plainKey = key.lstrip( "options." )
+		overrideKey = "renderSettingOverrides." + plainKey
+		# Assume the description is elem 1 in the key's list
+		overrideMetadata = [ "description", value[1] ]
+		overrideMetadata[1] = "Overrides the `{option}` render option:\n\n{description}".format(
+			option = plainKey,
+			description = overrideMetadata[1]
+		)
+		__overridesMetadata[overrideKey] = overrideMetadata
```

Comment:
If you wanted to make this a bit more readable, you could simplify this (from line 358) to:

```
__overridesMetadata[overrideKey] = [
    "description",
    "Overrides the `{option}` render option:\n\n{description}".format(
        option = plainKey,
        description = value[1]
    )
]

Reply:
eb0d6dcb98f7300fcde77d24def32010037d5d54